tests/coverage/reachable/bounds/reachable_fail/test.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+fn get(s: &[i16], index: usize) -> i16 {
+    s[index] // PARTIAL: `index = 15` causes failure, but `s[index]` is `COVERED`
```

Comment:
This is rather confusing, eh? I guess the return statement is unreachable.

Reply:
Yeah, sorry... you're right.